src/components/Form/Form.test.tsx

```diff
@@ -0,0 +1,51 @@
+// import { expect } from "chai";
```

Comment:
This should be updated eventually. It may be hard to write tests for layout-only components... but we can think of a way to test them.

Reply:
@EdwinGuzman Should I just remove this `test` file?  Or should I leave it in there with nothing in it?

Reply:
Just remove the entire file.

Reply:
I was able to write some tests for this.  The tests are fine, but I think they could be better.  For example, I was not able to get Enzyme's `parent()` method to work the way I wanted.